enclave/core/sgx/td_basic.c

```diff
@@ -18,6 +18,9 @@
 #include "linux/threadlocal.h"
 #endif
 
+void _oe_alloc_thread_teardown();
```

Comment:
There's just two, and they're only in two places now (malloc.c and snmalloc_wrapper.cpp), and a probably a single place eventually, so that's why I didn't do it. But I'm happy to do it.